GitUI/UserControls/FileStatusList.cs

```diff
@@ -902,6 +918,18 @@ private void SetFilterWatermarkLabelVisibility()
             FilterWatermarkLabel.Visible = FilterVisibleInternal && !FilterComboBox.Focused && string.IsNullOrEmpty(FilterComboBox.Text);
         }
 
+        private void FileStatusListLoading()
+        {
+            LoadingFiles.Visible = true;
+            NoFiles.Visible = false;
+            FilterVisibleInternal = false;
```

Comment:
Although hiding the `Filter` ComboBox is correct, I would not do it in order to reduce flicker.

Reply:
Changed this to only hide watermark. It looks weird when a filter is applied, but less flickering.

![image](https://user-images.githubusercontent.com/6248932/159378137-2d94eeca-b874-488b-81a2-756359b7778b.png)

Also fixed the prematurely submitted PR...

Reply:
Changed this as suggested in https://github.com/gitextensions/gitextensions/pull/9908#issuecomment-1079298052
Label now shown below the box